cli/src/cli.ts

```diff
@@ -273,6 +304,10 @@ export class CLI {
 		try {
 			logs.info("Disposing Kilo Code CLI...", "CLI")
 
+			const sessionService = SessionService.init()
```

Comment:
you can make this call only one time on the CLI constructor to remove some code duplication